solidity/.soliumrc.json

```diff
@@ -1,18 +1,16 @@
 {
-  "extends": "solium:recommended",
+  "extends": "keep",
   "plugins": [
     "security"
   ],
   "rules": {
-    "quotes": [
-      "error",
-      "double"
+    "security/no-block-members": [
+      "off",
+      [
+        "timestamp"
+      ]
     ],
-    "indentation": [
-      "error",
-      4
-    ],
-    "security/no-block-members": ["off", ["timestamp"]],
+    "indentation": "off",
```

Comment:
Maybe we shouldn't turn off indentation checks?

```suggestion
```

Reply:
This one actually behaves very strange. Initially it was not there, only when I copied the same config from `keep-ecdsa` I noticed why it might be turned off. When you turn it `on` and `npm run lint:sol` it will see that there are `.sol` files that need indentation changed. However, when you fix them manually (lint:fix:sol won't fix it) and run `npm run lint:fix:sol` again (ex. you have other changes that you want to fix) then it will revert you manual changes and running 'npm run lint:sol' will mark them as errors. So long story short.. `npm run lint:sol` indicates indentation errors made by `npm run lint:fix:sol`.

Reply:
Oh yeah, [I had the same problem in `keep-ecdsa`](https://github.com/keep-network/keep-ecdsa/commit/0befc5c9f84472c195a200cd13ced98f2d1d0ca6). We can turn off indentation check for solium as we have prettier running after solium and prettier will handle it.